### PR TITLE
 Changed the function name

### PR DESCRIPTION
**Pull request by Aman-Mandal**
Changed the function name in commandTest.cs file

### CODE REVIEW

File: test/Akkatecture.Tests/UnitTests/Commands/CommandTests.cs [hunk 1]
```diff
@@ -10,7 +10,7 @@ namespace Akkatecture.Tests.UnitTests.Commands
     public class CommandTests
     {
         [Fact]
-        public void InstantiatingCommand_WithValidInput_ThrowsException()
+        public void InstantiatingCommand_WithValidInput_ThrowsException_Succeeds()
```
Comment (malclear) [inline test/Akkatecture.Tests/UnitTests/Commands/CommandTests.cs:13]: Can you take out the "ThrowsExceptions" part of the name? There are none raised in this particular test.

**Review by malclear — CHANGES_REQUESTED** [review]

**Comment by malclear** [comment]
I've put a comment in your PR.
Thanks!!
*Malcolm Learner*

**Comment by Aman-Mandal** [comment]
So all I have to do is remove the "ThrowsException" from the name right sir?

**Comment by Aman-Mandal** [comment]
Done Sir .. !! Is this fine ?

**Pull request merged**